stories/documentation/forms/examples/html&css/form-basic.stories.ts

```diff
@@ -0,0 +1,159 @@
+import { Meta, StoryFn } from '@storybook/angular';
+
+interface FieldsetBasicStory {
+}
+
+export default {
+	title: 'Documentation/Forms/Examples/HTML&CSS',
+	argTypes: {
+	},
+} as Meta;
+
+function getTemplate(args: FieldsetBasicStory): string {
+	return `<form class="form">
+	<header class="form-header">
```

Comment:
I hadn't thought about it yesterday, but to put a `header`, you have to make sure there's sectioning content above it. `form` isn't one of them.